R/load_runner.R

```diff
@@ -41,10 +42,17 @@ run_load <- function(con, load, extracted_data, transformed_data, test_queries,
   invisible(TRUE)
 }
 
-do_load <- function(con, load, extracted_data, transformed_data, path, test_file, test_queries,
-                    log_table, log_data, dry_run) {
+do_load <- function(con, load, extracted_data, transformed_data, path,
+                    test_file, test_queries, pre_load, post_load, log_table,
+                    log_data, dry_run) {
   before <- test_queries(con)
```

Comment:
While these run I think we might want to indicate what it is up to perhaps? So that if an error happens the user knows if it's in the pre, main or post load?

Reply:
Added some more messages saying what is being run.